scripts/validate-requirements.sh

```diff
@@ -235,9 +235,9 @@ function echo_missing_installation () {
 # $3 = version to be displayed
 # $4 = web site to find the installation details of the missing binary
 function echo_wrong_version () {
```

Comment:
As we added a new parameter **$5**, it would be good to comment it's usage here as well.

Reply:
Good catch :)  
Fixed